config/crds/hive.openshift.io_clusterdeployments.yaml

```diff
@@ -26,7 +26,7 @@ spec:
     - jsonPath: .metadata.labels.hive\.openshift\.io/cluster-region
       name: Region
       type: string
-    - jsonPath: .metadata.labels.hive\.openshift\.io/version-major-minor-patch
```

Comment:
Are we completely sure no one uses this label before we remove it? From what I understand, the practice is to keep both and let the user decide which one they want to use

Reply:
We’re not removing the label, just changing which label we use in `oc get cd`

Reply:
To expand a bit: 99% of clusters (and 100% of prod clusters, since they should only be using GAed releases) will have the same value here before & after.